build.sbt

```diff
@@ -110,11 +115,8 @@ lazy val commonNativeSettings = Seq(
 
 lazy val coreSettings = commonSettings ++ publishSettings
 
-lazy val root = project.in(file("."))
```

Comment:
I removed this since it was not aggregating the `coreNative` module. This changes the name of the root module breaking some aliases I saw are defined in build.sbt. You can probably keep the root project if you add `aggregate(coreJVM, coreJS, coreNative)` but make sure to also add `settings(crossScalaVersions := List())`

Reply:
I think we should remove those aliases.

Reply:
Actually I think that was on purpose. Now opening `sbt` and running `test` fails locally, which is not cool for development.
So I think we should keep the root project as is (the only change I would do is rename it to `shapeless`).